web-ui/src/main/resources/catalog/locales/fr-admin.json

```diff
@@ -1022,5 +1023,6 @@
     "region/getmap/summaryWidth": "Summary width",
     "region/getmap/width": "Width",
     "metadata/editor": "Metadata editor configuration",
-    "metadata/editor/schemaConfig": "Configuration par standard"
+    "metadata/editor/schemaConfig": "Configuration par standard",
+    "logActivity" : "Activité"
```

Comment:
You can leave this here, but, we only need to update english translations in github. The translations will be popuplated from transifex (and then updated in github).